Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:13.14.0-alpine
+EXPOSE 3000 9229
```

Comment:
I'm unclear why 9229 is exposed but I'm not worried about it being a blocker to merging as this is a dev thing